### PR TITLE
Updated to Node 10 because 8 is being deprecated on Google app engine

### PR DESCRIPTION
**Pull request by stan29308**
If there's anything I missed just throw it up and I'll add it in

### CODE REVIEW

**Comment by sushshring** [comment]
Looks good. I'll let Travis build and verify that the code is still all compatible. Deploy a new version to App Engine staging manually and make sure that it all runs correctly.

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ branches:
   - master
   - dev
 node_js:
-- '8'
+- '10'
```
Comment (sushshring) [inline .travis.yml]: Ensure all the locations are changed - I don't know the exhaustive list.

**Comment by stan29308** [comment]
I don't know if it matters too much but Travis couldn't download the build cache.

File: .travis.yml (hunk 1 above)
Comment (stan29308) [reply .travis.yml]: As far as I know it should be all of them. I did a search for 'node' and '8'.

**Comment by stan29308** [comment]
Staging works

**Pull request merged**